docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml

```diff
@@ -128,5 +128,101 @@ required:
   - certifier
 example: |-
   {
-    "issue": "https://github.com/w3c-ccg/traceability-vocab/issues/238"
+    "id": "http://example.org/credentials/",
+    "type": [
+      "VerifiableCredential"
+    ],
+    "issuanceDate": "2021-02-04T20:29:37+00:00",
+    "issuer": "did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U",
+    "credentialSubject": {
+      "type": "USMCACertificateOfOrigin",
```

Comment:
should this type be actually above?

Reply:
Yes, true. I iterated a bit on this structure. This should have been moved back up. Fixing.